### DISCUSSION TITLE
8.1 Release

### DISCUSSION THREAD

**Discussion by shwethanidd** [VOLTTRON/volttron#2773] [General]
This release contains new features, fixes to issues reported by the community and the development team. The following list highlights the most notable changes included in 8.1:

- Historian related changes
   - Support for newer version of Mysql 8.0.
   - Updated SQL historians schema to combine topics and metadata into the same table. The new version of historian will work with existing tables (i.e. old schema) if tables already exists.
   - Updated SQL aggregate historian schema to store aggregates as double precision number instead of text – removed support for string aggregations such as GROUP_CONCAT.
- Ability to allow capabilities to RPCs dynamically at run time.
- Bug fixes for RabbitMQ federation and shovel. Added pytests for RabbitMQ federation and shovel.
- Updated tests for web authentication.
- Added testing of RabbitMQ VOLTTRON into Github actions/workflows.
- Video tutorial on overview of platform driver.
- Moved agents contributed by external community into 'services/core/contrib' folder, agents that may work but is not actively maintained by PNNL team into 'services/core/unsupported' and agents that are deprecated into 'deprecated' folder.

### Known Issues:
- RabbitMQ VOLTTRON does not work in Ubuntu 20.04 LTS
- There is a 6-7 compatibility issue when forwarding data from 6 instance to 7 instance

<hr /><em>This discussion was created from the release <a href='https://github.com/VOLTTRON/volttron/releases/tag/8.1'>8.1 Release</a>.</em>

**Comment by craigpnnl** [comment]
There is an issue with VC when configuring via vcfg.

The identity of the created agent is not what vc is expecting.   VC expects an identity of platform.historian, however vcfg doesn't default to an identity so it is installed with identity sqlhistorianagent-4.0.0_1.

**Reply by craigpnnl** [reply]
There is an issue with setuptools 40 in python 3.9.  This has been dealt with in #2780

**Reply by craigpnnl** [reply]
The issue with vcfg has been dealt with and will be in the soon to be released 8.1.1 #2782
